src/components/footer/Footer.scss

```
@@ -1,3 +1,98 @@
-div {
-  font-size: 15px;
+@mixin base($justify) {
+  display: flex;
+  justify-content: $justify;
+}
+
+.footer {
+  position: relative;
+  margin-top: 100px;
+  transform: translateY(0%);
+
+  .footerNav {
+    @include base(space-around);
+    width: 100%;
+    padding-top: 15px;
+    padding-bottom: 15px;
```

```suggestion
    padding: 15px 0;
```
따로 관리하는 것보단 이렇게 관리하는 편이 낫겠네요!

---

반영하여 수정하였습니다!